week2/velog/main.js

```diff
@@ -0,0 +1,86 @@
+const body = document.querySelector("body");
+
+const modalBackground = document.querySelector(".modal-background"),
+    modalBox = document.querySelector(".modal-box"),
+    modalBtn = document.querySelector(".modal-button");
+
+const cardSection = document.querySelector(".cards"),
+    cardArtics = document.querySelectorAll(".card");
+
+const slideBtns = document.querySelectorAll(".slide-button");
+
+const daySelect = document.querySelector(".js-select"),
+    optionBox = document.querySelector(".js-options"),
+    optionItems = document.querySelectorAll(".js-option");
+let selectedItem = document.querySelector(".option-selected");
+
+// For 드롭다운
+function handleOptionClick(e) {
+    // select 텍스트 변경
+    daySelect.value = e.currentTarget.innerHTML;
+
+    // 선택된 옵션 텍스트 색상 변경
+    selectedItem.classList.remove("option-selected");
+    e.currentTarget.classList.add("option-selected");
+    selectedItem = e.currentTarget;
+
+    // select box 숨기기
+    optionBox.classList.remove("select-selected");
+}
+
+function handleSelectClick() {
+    // select box 토글
+    optionBox.classList.contains("select-selected") ?
+        optionBox.classList.remove("select-selected") :
+        optionBox.classList.add("select-selected");
+}
```

Comment:
```suggestion
function handleSelectClick() {
    // select box 토글
        optionBox.classList.toggle("select-selected");
}
```

단순히 드롭다운 박스만 나왔다 없앴다 하는거는 이렇게 줄일수도 있을 것 같아!!

Reply:
헐 toggle이라는 게 있는 줄 전혀 몰랐다.. 땡큐입니다🙇‍♂️🙇‍♂️🙇‍♂️🙇‍♂️🙇‍♂️